analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RoslynSymbolicExecution.cs

```diff
@@ -211,21 +219,20 @@ context.Operation.Instance.Kind switch
                 _ => context.State
             };
 
-        private static ExceptionState ExceptionCandidate(IOperationWrapperSonar operation) =>
-            operation.Instance.Kind switch
+        private static ExceptionState ExceptionCandidate(IOperation operation) =>
```

Comment:
Please revert this, we should stick with the wrapper as the default type to pass around, so we are not tempted to create it. Some of the operation might need its properties

Reply:
When we pass `node.Block.BranchValue` we don't have a wrapper  so we had to create one just for calling this method that did not actually use it. Since now we have 2 different methods I can revert the changes done on this one.